src/pybind/mgr/nfs/export.py

```diff
@@ -656,6 +661,17 @@ def create_cephfs_export(self,
                              access_type: str,
                              clients: list = [],
                              sectype: Optional[List[str]] = None) -> Dict[str, Any]:
+
+        with open_filesystem(self.fs_client, fs_name) as fs_handle:
+            try:
+                stx = fs_handle.statx(path, cephfs.CEPH_STATX_MODE,
+                                      cephfs.AT_SYMLINK_NOFOLLOW)
```

Comment:
isn't a path containing a symlink not allowed to be exported in NFS ?
A chapter [Resolving symlinks in NFS](https://docstore.mik.ua/orelly/networking_2ndEd/nfs/ch06_04.htm) from a book helps a bit to gain understanding

Reply:
interesting data, the last arg in `statx` is used to set `AT_NO_ATTR_SYNC`/ `AT_SYMLINK_NOFOLLOW` flags, we don't need any of those here then. I'll remove it. thanks for pointing this out,

Reply:
removed `cephfs.AT_SYMLINK_NOFOLLOW`